src/frontend/jest/playwright.setup.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * `frontend/jest-playwright.config.js` is for configuring Playwright's launch config options
+ * `jest/playwright.setup.ts` is for configuring `jest`, `browser`, and `page` objects
+ */
+
+// (thuang): This is the max time a test can take to run.
+// Since when debugging, we run and !headless, this means
+// a test can take more time to finish, so we don't want
+// jest to shut off the test too soon
```

Comment:
Can you please clarify this a bit? 🙏

Reply:
Absolutely! So when you run a test with headful mode (which means Playwright will open a browser window for you to see what it is clicking) often times you will add `await page.waitForTimeout(30 * 1000)` at a certain step, so you have 30 seconds to inspect the browser's state and see why a step is not doing what you expect to do, etc.. 

But your extra 30 seconds still counts towards the max timeout a test is allowed to run, which by default is 30 seconds. So if it takes the test 10 seconds to get to the state you want to inspect, then you'll only have 20 seconds left to do so, even though your code sets `await page.waitForTimeout(30 * 1000)`

Thus we just expand the default timeout to 2 mins for each test, so you likely won't run into that problem. And if so, you know where to increase the timeout limit to a higher number!